docs/releases.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
We're not adding this page right?

Reply:
yes, it's disabled in sidebars.js. Will also delete in prod.